app/src/main/java/com/frinder/frinder/activity/DiscoverActivity.java

```diff
@@ -101,74 +114,109 @@ private void getCurrentLocation() {
         locationUtilInstance.startLocationUpdates(this,locationUtilsCallback);
     }
 
-    public void getNearbyUsers() {
+    public void getdiscoverUsers() {
         //Clear the list each time discover menu button is clicked as user list will change based on who is nearby
-        if (!users.isEmpty()) {
-            users.clear();
+        if (!nearbyUsers.isEmpty()) {
+            nearbyUsers.clear();
             adapter.notifyDataSetChanged();
         }
 
+        /*
         // ToDo Mallika - change userFirebaseDas.getAllUsers() to return users based on current filters
         userFirebaseDas.getAllUsers(new UserFirebaseDas.OnCompletionListener() {
             @Override
             public void onUsersReceived(ArrayList<User> users) {
                 readAllUsersComplete(users);
             }
         });
-    }
-
-
-
-    public void readAllUsersComplete(ArrayList<User> userList) {
-        Log.d(TAG, "in readAllUsersComplete");
-        Log.d(TAG, userList.toString());
+        */
 
-        //ToDo - transfer this logic to Firebase if possible
+        SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(this);
+        double searchRadius = pref.getInt("radius", 200);
 
-        ArrayList<User> nearbyUsers = new ArrayList<>();
-        ArrayList<Float> distanceFromCurrentUser = new ArrayList<>();
-        TreeSet<Float> distances = new TreeSet<>();
+        //get filterInterests
+        final ArrayList<String> filterInterests = new ArrayList<>();
```

Comment:
This is temporary, right?

Reply:
Yes, it is temporary. I will remove it when the filter UI is ready.